app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/ide.tsx

```diff
@@ -0,0 +1,74 @@
+/** @file  */
+import * as react from "react";
+
+import * as service from "../service";
+
+const IDE_CDN_URL = "https://ensocdn.s3.us-west-1.amazonaws.com/ide";
+
+interface Props {
+  project: service.Project;
+  backendService: service.Backend;
+}
+
+declare global {
+  const enso: { main: (config: object) => void; };
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const Ide = ({ project, backendService }: Props) => {
+  react.useEffect(() => {
+    void (async () => {
+      const ideVersions = await backendService.listVersions({
+        versionType: service.VersionType.ide,
+        default: true,
+      });
+      /* TODO: make latest nightlies configurable again */
+      /* TODO: reload style.css and index.js.gz when opening a different version */
+      const projectIdeVersion = "2023.1.1-nightly.2023.2.9"; // project.ideVersion?.number.value ?? ideVersions[0]!.number.value;
```

Comment:
1. hardcoded value
2. commented code

Reply:
intentional, as of right now the latest nightly doesn't accept parameters in `enso.main` - not sure what the correct solution would be. the commented code is the actual code that should be there, but it currently doesn't work